source/indexes.txt

```diff
@@ -22,4 +22,73 @@ Optimize Queries with Indexes
    :titlesonly:
    :maxdepth: 1
 
-   /work-with-indexes
\ No newline at end of file
+   /work-with-indexes
+
+Overview
+--------
+
+On this page, you can see copyable code examples that demonstrate how to work with common
+types of indexes that you can use with the {+driver-short+}.
+
+.. tip::
+
+  To learn more about working with indexes, see the :ref:`cpp-work-with-indexes`
+  guide. To learn more about any of the indexes shown on this page, see the link
+  provided in each section.
+
+To use an example from this page, copy the code example into the
+:ref:`sample application <cpp-index-sample>` or your own application.
+Be sure to replace all placeholders in the code examples, such as ``<connection string>``, with
+the relevant values for your MongoDB deployment.
+
+.. _cpp-index-sample:
+
+.. include:: /includes/usage-examples/sample-app-intro.rst
+
+.. literalinclude:: /includes/usage-examples/sample-app.cpp
+   :language: cpp
+   :copyable:
+   :linenos:
+   :emphasize-lines: 23-25
+
+Single-Field Index
+------------------
+
+The following code shows how to create an ascending single-field index:
+
+.. literalinclude:: /includes/usage-examples/index-code-examples.cpp
+   :start-after: start-single-field
+   :end-before: end-single-field
+   :language: cpp
+   :copyable:
+
+To learn more about single-field indexes, see the
+:ref:`cpp-single-field-index` guide.
```

Comment:
S: since this isn't a full guide, just a section of the "Work with Indexes" guide, I'd reword to something like:
```suggestion
:ref:`cpp-single-field-index` section of the Work with Indexes guide.
```

Reply:
Applies to all